### ISSUE TITLE
Classes should not include .

### ISSUE THREAD

**Issue by espenmn** [collective/collective.themefragments#2]
When using the mosaic tile, the html returns:
mosaic-collective.themefragments.fragment-tile
for class (the dots are in the html).

This makes it impossible to style the tile.

I have made a suggestion here:
https://github.com/espenmn/collective.themefragments/commit/9f2d13d392ee794b55ed0b406d7e54db7bea6381

**Comment by espenmn** [comment]
I discovered that it is possible to 'escape' the class, so actually it can be styled with:

`.mosaic-collective\.themefragments\.fragment-tile { width: 100%}`

**Comment by datakurre** [comment]
Yes. This is a "design feature" of Mosaic editor (from Deco editor) and not trivial to change in the current version, unfortunately. This applies to all tiles.
